app/search_builders/concerns/iiif_print/highlight_search_params.rb

```diff
@@ -6,9 +6,10 @@ module HighlightSearchParams
     def highlight_search_params(solr_parameters = {})
       return unless solr_parameters[:q] || solr_parameters[:all_fields]
       solr_parameters[:hl] = true
-      solr_parameters[:'hl.fl'] = 'all_text_tsimv'
+      solr_parameters[:'hl.fl'] = '*'
```

Comment:
Non-blocker: talk to me about this.

Reply:
This is suppose to add all indexed fields into hl.fl which has to do with highlighting.  I'm not too sure of how this works to be honest but it was something Rob told us to change.